store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/hdfs/mapper/SampleDataForSplitPointsMapper.java

```diff
@@ -63,7 +65,7 @@ protected void setup(final Context context) {
 
     @Override
     protected void map(final Element element, final Context context) throws IOException, InterruptedException {
-        if (Math.random() < proportionToSample) {
+        if (secureRandom.nextDouble() < proportionToSample) {
```

Comment:
Again this is just used to create samples out of the data there's no need for secure random.

Reply:
Can revert back makes sense